keyboards/funky40/keymaps/default/readme.md.md

```diff
@@ -0,0 +1,5 @@
+![Funky40 Layout Images](https://imgur.com/a/D13SsJB)
+
+# Default Funky40 Layout
```

Comment:
```suggestion
# Default Funky40 Layout

![](https://i.imgur.com/jxoCDqx.png)
![](https://i.imgur.com/3bBL52A.png)
```
These need to be actual images. Please also remove the extra `.md` from the filename.